src/FNA3D_Driver_Vulkan.c

```diff
@@ -6499,29 +6519,50 @@ static void VULKAN_SetBlendState(
 ) {
 	VulkanRenderer *renderer = (VulkanRenderer*) driverData;
 
-	SDL_memcpy(&renderer->blendState, blendState, sizeof(FNA3D_BlendState));
-
-	/* Dynamic state */
 	VULKAN_SetBlendFactor(
 		driverData,
 		&blendState->blendFactor
 	);
+
 	VULKAN_SetMultiSampleMask(
 		driverData,
 		blendState->multiSampleMask
 	);
+
+	SDL_memcpy(&renderer->blendState, blendState, sizeof(FNA3D_BlendState));
```

Comment:
Do we need to do an SDL_memcmp and request a new pipeline for changes here?

Reply:
the comparisons and pipeline checks happen in the above functions, not sure if memcmp is worth doing before the copy?

Reply:
Oh wait, I see, yes, this is an error actually